src/js/stores/managers/layout-manager.js

```diff
@@ -0,0 +1,309 @@
+"use strict";
+
+var _ = require('lodash');
+var hat = require('hat');
+var ReactGridUtils = require('react-grid-layout').utils;
+var LayoutComponentColumns = require('../../constants/layout-constants').ColumnTypes;
+
+class LayoutManager {
+    constructor() {
+        // APEP TODO set initial state
+        this.layout = [];
```

Comment:
We may wish to add in some default components like before

Reply:
The idea for the default components was to have a base layout that would make sense to a first-time user - to aid this I think that some sort of nice tooltip library can be implemented as a separate feature

Reply:
Agreed, will add into a commit after we are happy with it.